### DISCUSSION TITLE
how can i install github on window 10 32bit

### DISCUSSION THREAD

**Discussion by Ethclan** [community/community#60524] [Other Feature Feedback, Questions, & Ideas] [answered]
![GitHubDesktopSetup-x64 exe 7_10_2023 10_25_19 AM](https://github.com/community/community/assets/118100994/ff449e82-0d36-4116-b21a-66ce5c46a327)
GITHUB for desktop 32bit?

i tried all methods still does't find the right way , i begin thinking it not possible to download @github for desktop 32bit,

it only works one 64bit

**Comment by techiechap** [comment]
Am not sure what do you mean by github installation,  you can develop your code by initializing  your repo  and use git commands for further management , Visit the official Git website: Go to the Git website at https://git-scm.com/.

Download the Git installer: On the homepage, you'll find a "Download" button. Click on it to proceed.

Choose your operating system: Git supports various operating systems. Select the appropriate installer for your operating system. For Windows, you'll have an option to download Git for Windows. For macOS, you can download the installer for macOS. For Linux, you can find instructions specific to your distribution.

Run the installer: Once the download is complete, run the installer file you downloaded.

Configure the installation: During the installation process, you may be presented with various configuration options. You can typically leave the default settings as they are, but you have the option to customize the installation if desired.

Select components: You'll be prompted to select components to install. It's recommended to keep the default selections unless you have specific preferences.

Choose an editor: You'll have the option to select a default text editor for Git. You can choose your preferred editor, such as Visual Studio Code or Vim, or you can opt for the default editor provided with the Git installation.

Adjust the system's PATH environment variable (Windows only): If you're on Windows, you'll be asked to choose how Git should be integrated with the Windows Command Prompt. Select the desired option, typically "Use Git from Git Bash only," unless you have a specific reason to choose a different option.

Complete the installation: Proceed with the installation process, following any remaining prompts or instructions.

Verify the installation: Once the installation is complete, open a terminal or command prompt and run the following command to verify that Git is installed successfully:

**Reply by GhostOf0days** [reply]
I don't think he meant Git. There is Github Desktop.

**Reply by Ethclan** [reply]
yeah github desktop for 32bit

**Reply by techiechap** [reply]
my bad, i never imagined that, Thanks for correcting it

**Reply by Ethclan** [reply]
But, thank you. for tryin to help

**Reply by Saqlain-Asif-26** [reply]
it is also not working on 64 bit, i'm facing same problem in 64 bit window 10

**Comment by GhostOf0days** [comment]
You can download Github Desktop [here](https://desktop.github.com).

**Reply by Ethclan** [reply]
you see i want 32bit

**Reply by GhostOf0days** [reply, marked as answer]
Not available. It's deprecated.

**Reply by Ethclan** [reply]
damn, so i will use the web only.

**Reply by ghostinhershell** [reply]
Hi @Ethclan, thanks so much for reaching out in the GitHub Community 👋 

Just a heads up that I will be marking @GhostOf0days response as the answer here since, as they noted, you must have a 64-bit operating system to run GitHub Desktop: https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/overview/supported-operating-systems-for-github-desktop.

**Reply by playboyFafo247** [reply]
K

**Reply by siba-9439** [reply]
GitHub Desktop has dropped support for 32-bit systems. As of late 2021, GitHub Desktop no longer offers an official 32-bit installer on the main download page. The GitHub Desktop client now only supports 64-bit versions of Windows. Unfortunately, if you're running Windows 10 (32-bit), you will not be able to use GitHub Desktop as it is no longer compatible with 32-bit systems. Alternatively you can use Git which works in CLI and also you can use GitHub web interface. And if you really need a GUI interface you can use below application for windows 32 bit:

Sourcetree: https://www.sourcetreeapp.com/
TortoiseGit: https://tortoisegit.org/
GitKraken: https://www.gitkraken.com/

**Reply by krishnarathore31893-hash** [reply]
i am downloading github 64 bit that time they so me error so what i do can you give me some example how i solve its ?

**Reply by krishnarathore31893-hash** [reply]
<img width="445" height="251" alt="image" src="https://github.com/user-attachments/assets/ff80863b-a22e-49dd-a189-7c039b28749f" />

**Reply by krishnarathore31893-hash** [reply]
what i do for this give me some example

**Comment by cehfisher** [comment]
Hi @Ethclan - It looks like your issue was mistakenly added to the accessibility discussion board, but resolved via [this comment](https://github.com/orgs/community/discussions/60524#discussioncomment-6407054). @queenofcorgis could you please move this to the correct category and close it out? Thanks!

**Comment by 7alimboyev** [comment]
where can i find github desktop to install on widows 8 32 bit????
